ledger-api/grpc-definitions/com/daml/ledger/api/v1/update_service.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+import "com/daml/ledger/api/v1/ledger_offset.proto";
+import "com/daml/ledger/api/v1/transaction.proto";
+import "com/daml/ledger/api/v1/transaction_filter.proto";
+import "com/daml/ledger/api/v1/transfer.proto";
+
+option java_outer_classname = "UpdateServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Allows clients to read updates (transactions and transfers) from the ledger.
+service UpdateService {
+
+  // Read the ledger's filtered transaction stream and related transfers for a set of parties.
+  // For transactions it lists only creates and archives, but not other events.
```

Comment:
this seems somewhat redundant, the type already tells you this.

Reply:
yes, I tend to agree. This description follow the same what we have in transaction_service. I would suggest keep it alike (minimize difference). WDYT?

Reply:
ah didn't realize it was just copied, let's keep it then.